### DISCUSSION TITLE
How to create TOC for astrowind

### DISCUSSION THREAD

**Discussion by abdelkareemkobo** [arthelokyo/astrowind#350] [Q&A]
What is the proper way to add Table of contents into astrowind! 
I have search the web and found multiple libs but i want to here the answer for an experience developer.
Thanks in advance

**Comment by masheeeen** [comment]
Hey, can you please tell me which libraries you found to implement a table of contents?

**Comment by abhinavkakku** [comment]
Looking for same, hope Devs put a documentation for implementing ToC

**Comment by prototypa** [comment]
Hi @abdelkareemkobo, @masheeeen and @abhinavkakku 

Check _Astro_ documentation: https://docs.astro.build/en/guides/markdown-content/#markdown-plugins

```js
import { defineConfig } from 'astro/config';
import remarkToc from 'remark-toc';

export default defineConfig({
  markdown: {
    // Applied to .md and .mdx files
    remarkPlugins: [remarkToc],
  },
});
```

Plugin documentation: https://github.com/remarkjs/remark-toc

**Reply by abhinavkakku** [reply]
Thanks, worked
